### ISSUE TITLE
New Rule: consistent default imports

### ISSUE THREAD

**Issue by stronny** [eslint/eslint#18840]
### Rule details

disallow `import {default as name} from 'module'` syntax

### Related ECMAScript feature

modules

### What type of rule is this?

Suggests an alternate way of doing something

### Example code

```js
/* incorrect */
import {default as name} from 'module' // superfluous syntax

/* correct */
import name from 'module' // shorter
import {default as name1, name2} from 'module' // default isn't the only thing to import
```

### Why should this rule be in the core instead of a plugin?

I have to admit I am not nearly enough educated in eslint ecosystem. If you think it belongs elsewhere, it's really your call, not mine.

### Participation

- [ ] I am willing to submit a pull request to implement this rule.

**Comment by mdjermanovic** [comment]
Hi @stronny, thanks for the issue!

This rule already exists in [`eslint-plugin-import`](https://www.npmjs.com/package/eslint-plugin-import):

https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-named-default.md

Given that it's a stylistic rule that isn't related to new ECMAScript features, I think it doesn't qualify for inclusion in the core, per our [rules policies](https://eslint.org/blog/2020/05/changes-to-rules-policies/).
